helpers/migrate.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python3
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import subprocess
+import sys
+import shutil
+import re
+
+
+class GSuiteMigration:
+    """
+    Migrate the resources from a flat project factory to match the new
+    module structure created by the G Suite refactor.
+    """
+
+    CORE_PROJECT_FACTORY_SELECTORS = [
+        ("google_organization", "org"),
```

Comment:
`google_organization.org`  was extracted to the `google_organization` submodule as the logic around it was duplicated in `gsuite_enabled` and `core_project_factory`.

Reply:
I've dropped this from the refactored resource list, and since it's a data source it won't cause issues.